ui/src/views/connection/index.vue

```diff
@@ -26,7 +26,7 @@ import type { ContentType } from '@/types/modules/connection.type';
 const title = ref<string>('');
 const token = ref<string>('');
 const defaultProtocol = ref<string>('ssh');
-const contentType = ref<ContentType>('setting');
+const contentType = ref<ContentType>('');
 const showDrawer = ref<boolean>(false);
 
 const handleUpdateDrawer = (show: boolean, _title: string, _contentType: ContentType, _token?: string) => {
```

Comment:
There are no obvious differences between the two versions of this code snippet you provided. They appear to be quite similar except for changing "ref<string>()" to "ref<ContentType>())". No issues were found or suggested optimizations could be made based on information available before September 2021 (the current date is May 15, 2024).